backend/.env

```
@@ -0,0 +1,3 @@
+PORT=5134
+CORS_ORIGIN=http://localhost:5173
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CORS_ORIGIN key should go before the PORT key

(UnorderedKey)